src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java

```diff
@@ -71,9 +71,11 @@
 
   private List<AdminDto> buildAdminDtoList(List<UserType> userTypeList) {
     return userTypeList.stream().map(userType -> {
+      List<AttributeType> userAttributes = userType.getAttributes();
       AdminDto adminDto = new AdminDto();
       adminDto.setUsername(userType.getUsername());
-      adminDto.setFullName(getUserFullName(userType.getAttributes()));
+      adminDto.setFullName(getUserFullName(userAttributes));
+      adminDto.setEmail(getUserEmail(userAttributes));
```

Comment:
Not sure why we aren't using the mapper to do this.
@jayanta2018 do you know why? (I'm assuming because of the attribute list, we can resolve that if so).

Reply:
@Judge40 I considered mapper here when I added the user full name. But as you mentioned already attribute list was the issue. I could not figure out how to use mapper to that.

Reply:
Okay, I have a solution to use the mapper. Just need to resolve some issues in the tests and I'll push it.

Reply:
@jayanta2018 I've pushed those changes, could you review the PR including my changes please.